requirements.txt

```diff
@@ -2,6 +2,6 @@ talisker[gunicorn,flask,raven,prometheus]==0.14.3
 canonicalwebteam.flask_base==0.2.0
 canonicalwebteam.http==1.0.1
 canonicalwebteam.search==0.2.0
-canonicalwebteam.templatefinder==0.1.1
+git+git://github.com/nottrobin/canonicalwebteam.templatefinder@0.2.0-ubuntu-support#egg=canonicalwebteam.templatefinder
```

Comment:
Why not release to pypi?

Reply:
Actually I thought this is neater, so we don't pollute pypi with alpha releases unnecessarily